Linux/sgx/test_app/sgx_u.mk

```diff
@@ -81,7 +81,7 @@ endif
 App_Cpp_Files := $(UNTRUSTED_DIR)/TestApp.cpp
 App_Cpp_Objects := $(App_Cpp_Files:.cpp=.o)
 
-App_Include_Paths := -I$(UNTRUSTED_DIR) -I$(SGX_SDK_INC)
+App_Include_Paths := -I$(UNTRUSTED_DIR) -I$(SGX_SDK_INC) -I$(SGX_SDK)/femc/runner/includes
```

Comment:
Do we need a different strategy for finding the include files, for real builds?

Reply:
Yes. Will probably need to make femc binding public and have it downloaded in the setup script for this project.